content/200-orm/200-prisma-client/500-deployment/301-edge/450-deploy-to-cloudflare.mdx

```diff
@@ -0,0 +1,541 @@
+---
+title: 'Deploy to Cloudflare Workers & Pages'
+navTitle: Deploy to Cloudflare
+metaTitle: 'Deploy to Cloudflare Workers & Pages'
+metaDescription: 'Learn the things you need to know in order to deploy an app that uses Prisma Client for talking to a database to a Cloudflare Worker or to Cloudflare Pages.'
+tocDepth: 3
+preview: true
+---
+
+<TopBlock>
+
+This page covers everything you need to know to deploy an app with Prisma ORM to a [Cloudflare Worker](https://developers.cloudflare.com/workers/) or to [Cloudflare Pages](https://developers.cloudflare.com/pages).
+
+</TopBlock>
+
+## General considerations when deploying to Cloudflare Workers
+
+This section covers _general_ things you need to be aware of when deploying to Cloudflare Workers or Pages and are using Prisma ORM, regardless of the database provider you use.
+
+### Using an edge-compatible driver¡
+
+When deploying a Cloudflare Worker that uses Prisma ORM, you need to use an [edge-compatible driver](/orm/prisma-client/deployment/edge/overview#edge-compatibility-of-database-drivers) and its respective [driver adapter](/orm/overview/databases/database-drivers#driver-adapters) for Prisma ORM.
+
+> **Note**: [Prisma Accelerate](/accelerate) enables you to access _any_ database from _any_ edge function provider. No edge-compatible driver is necessary.
```

Comment:
Also move below the list of supported drivers, similar to Vercel page.

Reply:
I put this further above because it feels very noisy to have two block quotes right after another, which would be the case if we put this quote after the bullets...